test/statetest/statetest.cpp

```diff
@@ -88,6 +99,10 @@ int main(int argc, char* argv[])
             ->required()
             ->check(CLI::ExistingPath);
 
+        std::optional<std::string> test_name;
+        app.add_option("--name", test_name,
```

Comment:
The "name" is too generic. We should also suggest this is a test filter. Maybe actually go with pytest `-k` which stands for "keyword".

Reply:
In geth it is `--run <regexp>`, also quite generic

Reply:
`--run` is fine to me. it comes from golang standard testing library.